dashboard_documentation.md

```diff
@@ -7,20 +7,26 @@ Dashboard Documentation
 =======================
 {: .anchor}
 
-This documentation will provide step-by-step examples of how to perform
+The Solar Forecast Arbiter Dashboard is a web interface for managing solar
+observation and forecast data and for evaluating solar forecast accuracy.
+This documentation provides step-by-step examples of how to perform
 common activities on the Solar Forecast Arbiter dashboard. The Content menu
-can be used to navigate between each activity. Each section will include
+can be used to navigate between each activity. Each section includes
 instructions and screenshots.
 
 Be sure to read the [Getting Started](#getting-started) section for instructions
 on how to access the current version of the dashboard.
+Please see the [data model documentation](/datamodel/) for more details on
+how the Solar Forecast Arbiter organizes metadata (Sites, Observations, and Forecasts)
+and time series data.
+Tutorial materials are available at this [link](https://github.com/SolarArbiter/2019-Denver-Workshop).
```

Comment:
Only thing I see is that `[link]` here could be a descriptive name, perhaps this could change to 'Tutorial materials are available in our Denver Workshop repository. Not a big deal, but helps with accessibility and seo.